daniel/pass/ps.h

```diff
@@ -0,0 +1,68 @@
+#ifndef PS_H
+#define PS_H
+
+#include <limits.h>
+
+#define MAX_PID 419430
+#define MAX_COMMAND 4096
+#define MAX_PROC_PATH 30
+
+/**
+ * a struct representing a single process.
+ */
+typedef struct _Process {
+    char name[500];
+    int pid;
+    char command[MAX_COMMAND];
+    char path[PATH_MAX];
+} Process;
+
+typedef  struct  t_int_array {
+    int length;
+    int data[MAX_PID];
+} int_array;
+
+/**
+ * The function gets a pid as an argument and returns info about a certain process.
+ * @param pid the process id.
+ * @return a process struct describing it's properties.
+ */
+Process getProcessInfo(int pid);
```

Comment:
For all of these functions, I think you generally put out params after in params